test/entrypoint/srv_ddl.lua

```diff
@@ -188,7 +191,9 @@ package.preload['customers-storage'] = function()
                 box.space['_ddl_sharding_key']:update(space_name, {{'=', fieldno_sharding_key, sharding_key_def}})
             end)
             rawset(_G, 'set_sharding_func', function(space_name, fieldno_sharding_func, sharding_func_def)
```

Comment:
To be honest, I would've prefer merging https://github.com/tarantool/ddl/pull/98 first and then setting up a full integration test with honest `ddl.set_schema` (like I did in sharding reload). https://github.com/tarantool/crud/blob/a7a3413ba283a1b4f5c64ec1fa87c3ef13b9b6cc/test/entrypoint/srv_ddl_reload.lua#L156-L174

Reply:
Sounds logical, but I would like to have tests with inserting sharding func manually to the space `_ddl_sharding_func` as well. Let's merge https://github.com/tarantool/ddl/pull/98 and then add tests with `ddl` here.

Reply:
Now there are tests integrated with `ddl.set_schema`.

Reply:
Well, there is no any `ddl.set_schema` calls. As far as I understand, `ddl` isn't good with migrations, but the thing that I used in `srv_ddl_reload` seems close to a real ddl migration. I think it would be better to use the whole `set_sharding_func_name` implementation from `srv_ddl_reload`.

Reply:
`ddl.set_schema` call is here https://github.com/tarantool/crud/blob/90f0c714b7daf29240a71e3804a81d734b1f9aa7/test/entrypoint/srv_ddl.lua#L191

Reply:
`ddl.set_schema` is called without any sharding functions in schema and then we set up sharding functions later in test cases with `set_sharding_func` helper.

Reply:
So if you try to test this with older ddl without any vshard support, it still will be ok.

Reply:
The idea is that user will commonly use a single `ddl.set_schema` with vshard sharding functions in the beginning and then make crud calls. Since we build many different test cases here, it is hard to use the same approach without any reworks (and recreating a new Tarantool instance for each case), but using the same helpers as in `srv_ddl_reload`, in my opinion, is a bit much honest.

Reply:
Maybe I don't get your remark, but `ddl.set_schema` is called with `vshard` sharding functions in schema https://github.com/tarantool/crud/blob/90f0c714b7daf29240a71e3804a81d734b1f9aa7/test/entrypoint/srv_ddl.lua#L166-L170

Reply:
Oh, sorry, you are right. I looks like I've confused the approach in ddl and ddl_reload tests.